### PR TITLE
Update slides

### PR DESCRIPTION
**Pull request by christianvedels**
Updated slides for presentation in Sienna

### CODE REVIEW

File: Project_dissemination/HISCO Slides/Slides.Rmd [hunk 1]
```diff
@@ -87,16 +87,18 @@ xaringanExtra::use_progress_bar(color = "#808080", location = "top")
...
 ## *Automatic Occupational Standardization with OccCANINE*
-### Christian Møller-Dahl, Torben Johansen, Christian Vedel
-### University of Southern Denmark, HEDG
+]
 
-#### Twitter: @ChristianVedel,
-#### Email: christian-vs@sam.sdu.dk
-#### arXiv: [arxiv.org/abs/2402.13604](https://arxiv.org/abs/2402.13604)
-#### Updated `r Sys.Date()` 
+#### Christian Møller-Dahl, Torben Johansen, Christian Vedel
```
Comment (TorbenSDJohansen) [inline Project_dissemination/HISCO Slides/Slides.Rmd:95]: Remove "-" between "Møller-Dahl"

File: Project_dissemination/HISCO Slides/Slides.Rmd [hunk 2]
```diff
@@ -193,31 +188,73 @@ https://youtu.be/BF_oNe-sABQ?si=Ef1YrkK43Ln_IHJS
...
-
-.pull-right[
 .red[**Type B:**]
 ```{r echo=FALSE, message=FALSE, warning=FALSE, out.height="300px"}
 knitr::include_graphics("Figures/TypeB.webp")
 ```
 ]
 
+---
+## Thanks to data contributors
```
Comment (TorbenSDJohansen) [inline Project_dissemination/HISCO Slides/Slides.Rmd:222]: Seems at a glance 23 sources, similar to older version. Is this correct, i.e., are there any new sources we should remember to add?

File: Project_dissemination/HISCO Slides/Slides.Rmd [hunk 3]
```diff
@@ -668,6 +706,42 @@ $$
...
+.pull-right[
+### Development
+- New training data
+- Digit-based seq2seq architecture (predict one digit at a time)
+- More string augmentation:
+  + Adversarial training examples: Changing descriptions only slightly to make the model make wrong guesses
+  + Double translation 
+  + Random strings should always be '-1'
+- Testing 'finetunability' to OCC1950, OCCICEM, PSTI, ISCO68 
+- Two user end options: *'fast'* or *'good'*
```
Comment (TorbenSDJohansen) [inline Project_dissemination/HISCO Slides/Slides.Rmd:736]: Maybe replace "*good*" with "*advanced reasoning*" (inspired by o1-mini vs o1 naming) or something?

File: Project_dissemination/HISCO Slides/Slides.Rmd (hunk 1 above)
Comment (christianvedels) [reply Project_dissemination/HISCO Slides/Slides.Rmd:95]: Check

**Pull request merged**